project/standard/templates/prod-webpack.config.js

```diff
@@ -16,7 +16,8 @@ module.exports = require('./MapStore2/build/buildConfig')(
     {
         '__PROJECTNAME__': path.join(__dirname, "js", "app"),
         '__PROJECTNAME__-embedded': path.join(__dirname, "MapStore2", "web", "client", "product", "embedded"),
-        '__PROJECTNAME__-api': path.join(__dirname, "MapStore2", "web", "client", "product", "api")
+        '__PROJECTNAME__-api': path.join(__dirname, "MapStore2", "web", "client", "product", "api"),
+        'geostory-embedded': path.join(__dirname, "MapStore2", "web", "client", "product", "geostoryEmbedded")
```

Comment:
In the dashboard implementation we had to add a custom entry in the project that import the default pruduct entry. This was needed to be able to override the path of localConfig

see
https://github.com/geosolutions-it/MapStore2/blob/213fe57837d942ba66af2a68f659be97118b8d36/project/standard/templates/js/dashboardEmbedded.jsx 
https://github.com/geosolutions-it/MapStore2/blob/75c8631cae65ddf5660d211e6953f1e0bbecfcde/project/standard/templates/prod-webpack.config.js